ext/odbc/php_odbc.c

```diff
@@ -2095,32 +2095,55 @@ int odbc_sqlconnect(odbc_connection **conn, char *db, char *uid, char *pwd, int
 		/* a connection string may have = but not ; - i.e. "DSN=PHP" */
 		if (strstr((char*)db, "=")) {
 			direct = 1;
+
+			size_t db_len = strlen(db);
```

Comment:
Add a comment above that this should be identical to the code in the PDO driver and vice versa

Reply:
Added in https://github.com/php/php-src/pull/12306/commits/ad06797547a8894d1e9889a4d005a8c6964e02d1